mypy/fastparse.py

```diff
@@ -1094,7 +1177,7 @@ def invalid_type(self, node: AST, note: Optional[str] = None) -> RawExpressionTy
     def visit(self, node: ast3.expr) -> Type: ...
 
     @overload  # noqa
-    def visit(self, node: Optional[AST]) -> Optional[Type]: ...
+    def visit(self, node: Optional[AST]) -> Optional[Type]: ...  # noqa
```

Comment:
It's sad this `#noqa` is still needed. The issue https://github.com/PyCQA/pyflakes/issues/434 was fixed two weeks ago, so we will be able to update `flake8` and remove these soon.